plugins/chain_api_plugin/chain.swagger.yaml

```diff
@@ -488,10 +494,25 @@ paths:
                   description: Type of key specified by index_position (for example - `uint64_t` or `name`)
                 encode_type:
                   type: string
-                upper_bound:
-                  type: string
-                lower_bound:
-                  type: string
+                  lower_bound:
+                    type: string
+                    description: Filters results to return the first element that is not less than provided value in set
+                  upper_bound:
+                    type: string
+                    description: Filters results to return the first element that is greater than provided value in set
```

Comment:
These two properties should be intended two spaces left

Reply:
Ended up making the changed and pushing